docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -0,0 +1,641 @@
+      - [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation).
+      - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+      - [Terraform CLI](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) v1.4.0 or greater, if you choose to follow along with the Terraform workflow.
+
+## Authenticate with Palette
+
+The initial step to deploy a PCG using Palette CLI involves authenticating with your Palette environment using the
+[`palette login`](../../palette-cli/commands/login.md) command.
+
+In your terminal, execute the following command.
+
+```bash
+palette login
+```
+
+Once issued, you will be prompted for several parameters to complete the authentication. The table below outlines the
+required parameters along with the values that will be utilized in this tutorial. If a parameter is specific to your
+environment and Palette account, such as your Palette API key, ensure to input the value according to your environment.
+Access the [Deploy a PCG to VMware vSphere](../pcg/deploy-pcg/vmware.md) guide for detailed information about each
+option.
+
+| **Parameter**                  | **Value**                                                                                                        | **Environment-Specific** |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| **Spectro Cloud Console**      | `https://console.spectrocloud.com`. If using a self-hosted instance of Palette, enter the URL for that instance. | No                       |
+| **Allow Insecure Connection**  | `Y`                                                                                                              | No                       |
+| **Spectro Cloud API Key**      | Enter your Palette API Key.                                                                                      | Yes                      |
+| **Spectro Cloud Organization** | Select your Palette Organization name.                                                                           | Yes                      |
+| **Spectro Cloud Project**      | `None (TenantAdmin)`                                                                                             | No                       |
+| **Acknowledge**                | `Y`                                                                                                              | No                       |
+
+After accepting the login banner message, you will receive the following output confirming you have successfull
+authentication with Palette.
+
+```text hideClipboard
+Welcome to Spectro Cloud Palette
+```
+
+The video below demonstrates Palette's authentication process. Ensure you utilize values appropriate for your
+environment.
+
+<Video title="palette-login-video" src="/videos/palette-login.mp4"></Video>
```

Comment:
This video displays `cardenas88karl-gmail` can we get some blurring on it?

Reply:
Using the prod-automation account still displays `cardenas88karl-gmail` when selecting the org.
![CleanShot 2024-03-13 at 10 17 39](https://github.com/spectrocloud/librarium/assets/63267693/d424b703-6cb5-4efa-9c26-6f430bea0e2d)
I increased the playback speed and used the stage account instead. What do you think?